library/std/src/backtrace.rs

```diff
@@ -476,3 +511,24 @@ impl RawFrame {
         }
     }
 }
+
+#[unstable(feature = "backtrace_frames", issue = "79676")]
+impl<'a> AsRef<[BacktraceFrame]> for Frames<'a> {
```

Comment:
This would be incompatible with lazy resolving of backtraces. Only an Iterator would be compatible.